packages/workflow/src/internals.ts

```diff
@@ -416,6 +418,8 @@ export class Activator implements ActivationHandler {
 
   public readonly registeredActivityNames: Set<string>;
 
+  public versioningBehavior?: VersioningBehavior;
```

Comment:
How do dynamic workflows pick their versioning behaviour per type?

Reply:
They can do it like is shown here: https://github.com/temporalio/sdk-typescript/pull/1679/files#diff-5ae1c5b6523b9493d1f852e9d303621ffdc8f817f1b80f38f572008572aec6feR14

In Python Chad didn't want to have a dynamic getter like we added in Java unless someone asks, which I can get behind, so I figured we'd do the same here.

Reply:
Not sure it makes sense to cut features from some SDKs, but not other

Reply:
I think the getter is a touch error prone. We could remove it from Java too, if no one ends up needing it in these other places. I didn't want to hold up being able to finish the Core ones over that, given these can get changed later anyway. Chad is who really cared: https://github.com/temporalio/sdk-python/pull/821#discussion_r2037340078

Reply:
I think the getter is a basic requirement since versioning a per workflow type option per the design doc. I am just not sure why other SDKs users have to have reduced functionality.

Reply:
I'm not the one who didn't want to add it :man_shrugging: . I agree with the principle. In practice, I suspect most people doing dynamic workflows would be fine with one behavior. It's hard to say. I'm fine either way, I think Chad is who you have to convince.

Reply:
Has the getter now